src/extractor/scripting_environment_lua.cpp

```diff
@@ -1171,14 +1163,14 @@ void LuaScriptingContext::ProcessNode(const osmium::Node &node,
     {
     case 4:
     case 3:
-        node_function(profile_table, node, result, relations);
```

Comment:
Otherwise it complained that `Node` has no copy constructor and it cannot copy it.